target/sdk/files/Makefile

```diff
@@ -14,7 +14,7 @@ export TOPDIR LC_ALL LANG SDK
 
 world:
 
-DISTRO_PKG_CONFIG:=$(shell $(TOPDIR)/scripts/command_all.sh pkg-config | grep -E '\/usr' -m 1)
```

Comment:
Should we drop the `-E` ?

Reply:
`-E` is for *extended regular expression* according to man, so we don't need it.

Reply:
yep that was the idea... no idea why it was used on the first place?